ecommerce/extensions/checkout/views.py

```diff
@@ -195,7 +195,7 @@ def get_context_data(self, **kwargs):  # pylint: disable=arguments-differ
             'disable_back_button': self.request.GET.get('disable_back_button', 0),
         })
         context.update({
-            'ar_name': self.get_extended_field(self.request.user.account_details(self.request), 'arabic_name')
+            'ar_name': self.get_extended_field(self.request.user.account_details(self.request), 'arabic_name', '')
```

Comment:
@alfredchavez, I am wondering if it makes sense to return the full name instead? CC @giovannicimolin

Reply:
@lgp171188  mmm, yes, I was also considering it, I am going to change it to return the full_name in case arabic_name is not defined
CC @giovannicimolin

Reply:
@alfredchavez Yup, full name makes sense when there's no Arabic name available.

Reply:
@lgp171188 I've changed it so it uses the order user full name instead of arabic name in case it is not available, 
also fixed a "bug" that I just noticed, I've been using the current request user for the invoice instead of the order user, which I have now fixed, let me know if more change is needed here 🙂

Reply:
@lgp171188 I've changed it so it uses the order user full name instead of arabic name in case it is not available, 
also fixed a "bug" that I just noticed, I've been using the current request user for the invoice instead of the order user, which I have now fixed, let me know if more change is needed here 🙂 CC @giovannicimolin